RHEL/6/input/checks/file_permissions_ungroupowned.xml

```diff
@@ -16,9 +15,19 @@
   <unix:file_test xmlns:unix="http://oval.mitre.org/XMLSchema/oval-definitions-5#unix"
   check="all" comment="files with no group owner" check_existence="none_exist"
   id="test_file_permissions_ungroupowned" version="1">
+    <!-- OVAL 5.11 rc-2 within the effort to add oval-def:NotesType construct also to variables
+         moved NotesType construct to the oval-common-schema:
+         [1] https://github.com/OVALProject/Language/issues/6
+         [2] https://github.com/OVALProject/Language/commit/c34abb871ee446d9fd9e50a8ccbc8a8e2af5db20
+         But this change broke backward-compatibility as reported in:
+         [3] https://github.com/OVALProject/Language/issues/237
+         [4] http://making-security-measurable.1364806.n2.nabble.com/5-11-backwards-incompatibility-issue-Suspect-the-fix-for-issue-6-was-the-culprit-tp7586190.html
+         Therefore as a temporary workaround to have both OVAL 5.10 and also OVAL 5.11 valid content,
+         temporarily comment out the <notes> construct below till the issue [3] is fixed in OVAL 5.11.1
+         version.
     <notes>
       <note>This will enumerate all files on local partitions</note>
-    </notes>
+    </notes> -->
```

Comment:
I don't understand why the <notes> element is commented by the patch. Is that intentional?